docs/END_USER_DOCUMENT.md

```diff
@@ -750,3 +751,38 @@ AsyncBooksAppClient asyncClient = MoccaClient.Builder
         .client(executorClient)
         .build(AsyncBooksAppClient.class);
 ```
+## 8. Request Validation
+Mocca now supports validation of request parameters using a standard bean validation 2.0 implementation like hibernate.
+Please refer to [this site](https://beanvalidation.org/2.0-jsr380/) for information on bean validation. 
+
+**Important Note:** Mocca supports Bean Validation 2.0, not Jakarta Bean Validation 2.0 which has repackaged the api 
+classes from `java.validation` to `jakarta.validation`.
+
+### 8.1 Adding Request Validation to your Application
+If an implementation of the bean validation-api is provided on the classpath, bean validation will be automatically
```

Comment:
We also need to mention that Bean Validation annotations are ignored if set to variables set as `@Var(raw = true)`

Reply:
that's not entirely true. Annotations on the method itself will be honored.

Reply:
Do we need to add this exception? I think it still makes sense to validate method parameters, even ones marked "raw", especially if it's marked `@NotNull`

Reply:
I see. Ok, let's leave it like that then. Thanks for clarifying it.